core/domain/stats_jobs_one_off.py

```diff
@@ -138,6 +138,14 @@ def map(item):
                 if answer['classification_categorization'] != (
                         exp_domain.DEFAULT_OUTCOME_CLASSIFICATION):
                     useful_feedback_count += 1
+                else:
+                    exploration = exp_services.get_exploration_by_id(
```

Comment:
I've done this inside a separate 'else' branch because we should be using GET ops only when they're really required. For us to calculate useful feedback count, only one of the two conditions needs to be true. So, if the first condition is True, on point in doing the GET op for the exploration and checking the second condition.

Reply:
That makes sense; I agree. Thanks!

Reply:
That said, I'm not sure I follow the logic here.

If we reach the else branch, then this means that we're in "default outcome" territory. So why are you reading from the answer groups field? Shouldn't you be checking the default outcome field instead?

Reply:
Oh, thats right. Done.